src/main/java/seedu/address/ui/PersonCard.java

```diff
@@ -59,6 +65,21 @@ public PersonCard(Person person, int displayedIndex) {
         address.setText(person.getAddress().value);
         email.setText(person.getEmail().value);
         birthday.setText(person.getBirthday().toUi());
+        String goalText;
+        if (person.getGoal().getFrequency().equals(Goal.Frequency.NONE)) {
```

Comment:
`person.getGoal().isNoneFrequency()` would be better. Can hide the implementation away from UI